flax/configurations.py

```diff
@@ -70,6 +70,21 @@ def __repr__(self):
     values_repr = ', '.join(f'\n  {k}={v!r}' for k, v in self._values.items())
     return f'Config({values_repr}\n)'
 
+  @contextmanager
+  def temp_flip_flag(self, var_name: str, var_value: bool):
```

Comment:
`Config` should inherit from `threading.local` to make this safe.

Reply:
Done.